lib/backend.js

```diff
@@ -553,7 +555,7 @@ exports.createServer = (dynamodb, docClient) => {
       })
   }))
 
-  app.delete('/tables/:TableName/items/:key', asyncMiddleware((req, res) => {
+  app.delete('/tables/:TableName/*items/:key', asyncMiddleware((req, res) => {
```

Comment:
Why this change?

Reply:
In raw page, the delete requests are send as `DELETE /tables/:TableName/raw-items/:key`.